test/variables/ref/errors-type-mismatch-fn-2.chpl

```diff
@@ -0,0 +1,11 @@
+var global: int;
```

Comment:
Looks like you don't use global, can it be removed?  I know that will impact the line numbers for your output file